### ISSUE TITLE
ability to identify and report on multiple subjects in the image

### ISSUE THREAD

**Issue by QuantumPhilosophy** [QuantumPhilosophy/anonymous-pumpkin#21]
### User Stories
**Nice-To-Have:** Ability to identify and report on multiple subjects in the image

### Tasks
- [ ] Research GoogleCloudVision API Docs to see if multiple subjects within images are returned
- [ ] Update `wikipedia-helper.js` to accept an array of search topics

### Issue Links
**Depends On:** API helper setup for GoogleCloudVision #16 and Wikipedia #17
